tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -558,86 +571,98 @@ SpirvEvalInfo SPIRVEmitter::doDeclRefExpr(const DeclRefExpr *expr) {
 }
 
 SpirvEvalInfo SPIRVEmitter::doExpr(const Expr *expr) {
+  SpirvEvalInfo result(/*id*/ 0);
+
+  const bool literalType = isLiteralType(expr->getType(), astContext);
```

Comment:
Nit: `isNonLiteralType = !...` so that we don't need to negate twice in the following.

Reply:
Good idea